### ISSUE TITLE
Cannot fund carrier accounts

### ISSUE THREAD

**Issue by FenceDad** [ShipEngine/shipengine-dotnet#55]
I am unable to fund carrier accounts as we have in the past.   Is this available to the new API?

Thank you!

**Comment by AnneOReilly** [comment]
@FenceDad Can you share some more context around the issue you're having adding funds?  We may need to open you a ticket with our support team to dig in.

**Comment by brprigge** [comment]
@AnneOReilly I think he means it is just not in the SDK. We turned on auto funding to get around it, but it is near the top of my list of endpoints to implement.

**Comment by FenceDad** [comment]
@brprigge, you are correct.  We can work around it by pushing to the API directly since it is a simple call.  Thank you for the quick response!
